Test failure: quarkus-integration-test-logging-gelf fails with podman

### Describe the bug

I am working on a CI run which uses podman, and it has shown several failing tests. https://github.com/holly-cummins/quarkus/actions/runs/4844611295/jobs/8632987478 is an example run with failures. https://github.com/holly-cummins/quarkus/tree/podman-build has the (messy) workflow code.

Note that I’ve disabled this test when the CI is running with Podman, to keep the build clean. As part of the fix for this, the `IS_PODMAN` guard should be removed from the `pom.xml`. 

### Actual behavior

```
2023-04-30T17:57:00.0138360Z [INFO] Compiling 2 source files with javac [debug deprecation release 11] to target/test-classes
2023-04-30T17:57:00.2033528Z [INFO] 
2023-04-30T17:57:00.2034845Z [INFO] --- docker-maven-plugin:0.42.1:stop (docker-start) @ quarkus-integration-test-logging-gelf ---
2023-04-30T17:57:01.0469549Z [INFO] 
2023-04-30T17:57:01.0470852Z [INFO] --- docker-maven-plugin:0.42.1:start (docker-start) @ quarkus-integration-test-logging-gelf ---
2023-04-30T17:57:20.1073073Z [INFO] DOCKER> Pulled docker.io/elastic/logstash:7.16.3 in 18 seconds 
2023-04-30T17:57:20.1363289Z [INFO] DOCKER> [docker.io/elastic/elasticsearch:7.16.3] "elasticsearch": Start container bae3bcf9c63f
2023-04-30T17:57:21.6211939Z [ERROR] DOCKER> Error occurred during container startup, shutting down...
2023-04-30T17:57:21.6213657Z [ERROR] DOCKER> I/O Error [Unable to start container id [bae3bcf9c63f] : {"cause":"OCI runtime error","message":"crun: the requested cgroup controller `cpu` is not available: OCI runtime error","response":500} (Internal Server Error: 500)]
2023-04-30T17:57:21.6215594Z [INFO] 
2023-05-02T15:46:29.6816528Z [ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.42.1:start (docker-start) on project quarkus-integration-test-logging-gelf: I/O Error: Unable to start container id [b44bdc26ab68] : {"cause":"OCI runtime error","message":"crun: the requested cgroup controller `cpu` is not available: OCI runtime error","response":500} (Internal Server Error: 500) -> [Help 2]
2023-05-02T15:46:29.6817795Z org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.fabric8:docker-maven-plugin:0.42.1:start (docker-start) on project quarkus-integration-test-logging-gelf: I/O Error
2023-05-02T15:46:29.6818526Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:375)
2023-05-02T15:46:29.6819217Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:351)
2023-05-02T15:46:29.6819769Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
2023-05-02T15:46:29.6820311Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:171)
2023-05-02T15:46:29.6820912Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:163)
2023-05-02T15:46:29.6821546Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2023-05-02T15:46:29.6822352Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2023-05-02T15:46:29.6823342Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2023-05-02T15:46:29.6824056Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute 
```
### How to Reproduce?

Reproduction note: I haven't been able to reproduce this locally on mac with podman 4.03.
With the latest mac version of podman `4.4.4`, I get a different failure, 

```
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.42.1:start (docker-start) on project quarkus-integration-test-logging-gelf: I/O Error: Unable to create container for [docker.io/elastic/logstash:7.16.3] : {"cause":"image not known","message":"no such image: docker.io/elastic/logstash:7.16.3: image not known","response":404} (Not Found: 404) -> [Help 1]
```

Picking up my branch to reproduce on CI should work, or running locally on linux with a podman engine of `4.5`. 


## Environment Info 
```
2023-05-02T10:44:48.4887010Z ##[group]Run docker version
2023-05-02T10:44:48.4887318Z [36;1mdocker version[0m
2023-05-02T10:44:48.4949771Z shell: /usr/bin/bash -e {0}
2023-05-02T10:44:48.4950031Z env:
2023-05-02T10:44:48.4950252Z   LANG: en_US.UTF-8
2023-05-02T10:44:48.4950613Z   COMMON_MAVEN_ARGS: -e -B --settings .github/mvn-settings.xml --fail-at-end
2023-05-02T10:44:48.4951271Z   NATIVE_TEST_MAVEN_ARGS: -Dtest-containers -Dstart-containers -Dquarkus.native.native-image-xmx=5g -Dnative -Dnative.surefire.skip -Dformat.skip -Dno-descriptor-tests install -DskipDocs
2023-05-02T10:44:48.4952041Z   JVM_TEST_MAVEN_ARGS: -Dtest-containers -Dstart-containers -Dformat.skip -DskipDocs -Dquarkus.test.hang-detection-timeout=60
2023-05-02T10:44:48.4952507Z   DB_USER: hibernate_orm_test
2023-05-02T10:44:48.4952790Z   DB_PASSWORD: hibernate_orm_test
2023-05-02T10:44:48.4953053Z   DB_NAME: hibernate_orm_test
2023-05-02T10:44:48.4953388Z   IS_PODMAN: true
2023-05-02T10:44:48.4953637Z   MAVEN_OPTS: -Xmx2g -XX:MaxMetaspaceSize=1g
2023-05-02T10:44:48.4954244Z   JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.6-10/x64
2023-05-02T10:44:48.4954619Z   JAVA_HOME_17_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.6-10/x64
2023-05-02T10:44:48.4954970Z   DOCKER_HOST: unix:///run/user/1001/podman/podman.sock
2023-05-02T10:44:48.4955210Z ##[endgroup]
2023-05-02T10:44:49.5351123Z Client:
2023-05-02T10:44:49.5351708Z  Version:           20.10.24+azure-1
2023-05-02T10:44:49.5351963Z  API version:       1.41
2023-05-02T10:44:49.5352176Z  Go version:        go1.19.6
2023-05-02T10:44:49.5352453Z  Git commit:        297e1284d3bd092e9bc96076c3ddc4bb33f8c7ab
2023-05-02T10:44:49.5352738Z  Built:             Fri Mar 31 18:34:38 UTC 2023
2023-05-02T10:44:49.5352965Z  OS/Arch:           linux/amd64
2023-05-02T10:44:49.5353186Z  Context:           default
2023-05-02T10:44:49.5353414Z  Experimental:      true
2023-05-02T10:44:49.5353560Z 
2023-05-02T10:44:49.5353709Z Server: linux/amd64/ubuntu-22.04
2023-05-02T10:44:49.5353969Z  Podman Engine:
2023-05-02T10:44:49.5354202Z   Version:          4.5.0
2023-05-02T10:44:49.5354439Z   APIVersion:       4.5.0
2023-05-02T10:44:49.5354636Z   Arch:             amd64
2023-05-02T10:44:49.5355016Z   BuildTime:        1970-01-01T00:00:00Z
2023-05-02T10:44:49.5355260Z   Experimental:     false
2023-05-02T10:44:49.5355459Z   GitCommit:        
2023-05-02T10:44:49.5355677Z   GoVersion:        go1.18.1
2023-05-02T10:44:49.5355968Z   KernelVersion:    5.15.0-1036-azure
2023-05-02T10:44:49.5356195Z   MinAPIVersion:    4.0.0
2023-05-02T10:44:49.5356410Z   Os:               linux
2023-05-02T10:44:49.5356604Z  Conmon:
2023-05-02T10:44:49.5356824Z   Version:          conmon version 2.1.7, commit: 
2023-05-02T10:44:49.5357197Z   Package:          conmon_2:2.1.7-0ubuntu22.04+obs15.24_amd64
2023-05-02T10:44:49.5357457Z  OCI Runtime (crun):
2023-05-02T10:44:49.5357678Z   Version:          crun version 1.8.4
2023-05-02T10:44:49.5357962Z commit: 5a8fa99a5e41facba2eda4af12fa26313918805b
2023-05-02T10:44:49.5358237Z rundir: /run/user/1001/crun
2023-05-02T10:44:49.5358444Z spec: 1.0.0
2023-05-02T10:44:49.5358706Z +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
2023-05-02T10:44:49.5359097Z   Package:          crun_101:1.8.4-0ubuntu22.04+obs55.10_amd64
2023-05-02T10:44:49.5359334Z  Engine:
2023-05-02T10:44:49.5359516Z   Version:          4.5.0
2023-05-02T10:44:49.5359765Z   API version:      1.41 (minimum version 1.24)
2023-05-02T10:44:49.5360012Z   Go version:       go1.18.1
2023-05-02T10:44:49.5360212Z   Git commit:       
2023-05-02T10:44:49.5360442Z   Built:            Thu Jan  1 00:00:00 1970
2023-05-02T10:44:49.5360673Z   OS/Arch:          linux/amd64
2023-05-02T10:44:49.5360884Z   Experimental:     false

--- a/integration-tests/logging-gelf/pom.xml
+++ b/integration-tests/logging-gelf/pom.xml
@@ -199,7 +199,7 @@
                                         <log>
                                             <prefix>Logstash:</prefix>
                                             <date>default</date>
-                                            <color>cyan</color>
+                                            <color>magenta</color>
                                         </log>
                                         <volumes>
                                             <bind>
@@ -213,7 +213,7 @@
                                                 <status>200</status>
                                             </http>
                                             <!-- Starting the pipeline can be long as it needs to communicate with Elasticsearch -->
-                                            <time>45000</time>
+                                            <time>90000</time>
                                         </wait>
                                         <dependsOn>
                                             <container>elasticsearch</container>

